R/1.0_module_data_summary.R

```diff
@@ -85,7 +85,7 @@ srv_data_summary <- function(id, teal_data) {
             obs_str_summary = ifelse(
               !is.na(obs),
               sprintf("%s/%s", obs_filtered, obs),
-              ""
+              ifelse(!is.na(obs_filtered), sprintf("%s/(new)", obs_filtered), "")
```

Comment:
@m7pr what do you think of this for new datasets?

Maybe just keep `1995`?

![image](https://github.com/user-attachments/assets/c4fb483a-8060-4e60-b132-10ed05e5a957)

Reply:
Sorry @averissimo just seeing this now. Is it possible to have a filter for ANL? if yes, then maybe 1995/1995? if it's not possible to set filter for ANL I would go with 1995